docs/mor/modules/ROOT/pages/taylor-green-vortex/index.adoc

```diff
@@ -0,0 +1,202 @@
+= 2D Advection-diffusion with Taylor-Green Vortex
+Hassan Ballout, Luca Berti, Christophe Prud'homme
+:page-tags: case
+:page-illustration: taylor-green-vortex/combined_figure.png
+:description: We simulate the 2D Taylor-Green vortex.
+:stem: latexmath
+:page-jupyter: true
+:jupyter-language-name: python
+:jupyter-language-version: 3.8
+
+
+== Introduction
+
+We study the solution of a 2D advection-diffusion equation (used to model the dispersion of a contaminant) in presence of a Taylor-Green vortex stem:[\beta(x,t)]. The problem can be solved for different values of the diffusion coefficient stem:[\mu].
```

Comment:
We study the solution of a 2D advection-diffusion equation used to model the dispersion of a contaminant  with 2D Taylor-Green vortex stem:[\beta(x,t)] as advection field. The problem can be solved for different values of the diffusion coefficient stem:[\mu].